pymtl3/stdlib/rtl/enrdy_queues_test.py

```diff
@@ -9,7 +9,7 @@
 import pytest
 
 from pymtl3 import *
-from pymtl3.stdlib.test import TestSinkCL, TestSrcCL
+from pymtl3.stdlib.test import TestSinkCL, TestSrcCL, run_sim
```

Comment:
Is the idea to prefer using `stdlib.run_sim` over customized ticking method? That sounds like a huge improvement

Reply:
I think the problem is we had too many run_sim with basically the same code ... Now I just remove the redundant ones. I already did that in pymtl3-net.